File: accelerator-core/src/main/java/com/opentok/accelerator/textchat/utils/TestUtils.java
package com.opentok.accelerator.textchat.utils;

import java.util.Random;

public class TestUtils {

    public static String generateString(int length){

        StringBuilder tmp = new StringBuilder();
        for (char ch = '0'; ch <= '9'; ++ch)
            tmp.append(ch);
        for (char ch = 'a'; ch <= 'z'; ++ch)
            tmp.append(ch);
        char[] symbols = tmp.toString().toCharArray();
        char[] buf = new char[length];
        Random random = new Random();

        for (int idx = 0; idx < buf.length; ++idx)
            buf[idx] = symbols[random.nextInt(symbols.length)];
        return new String(buf);
    }
}

File: accelerator-core/src/androidTest/java/com/opentok/accelerator/utils/TestUtils.kt
package com.opentok.accelerator.utils

import java.util.Random

object TestUtils {
    fun generateString(length: Int): String {
        val tmp = StringBuilder()
        run {
            var ch = '0'
            while (ch <= '9') {
                tmp.append(ch)
                ++ch
            }
        }
        var ch = 'a'
        while (ch <= 'z') {
            tmp.append(ch)
            ++ch
        }
        val symbols = tmp.toString().toCharArray()
        val buf = CharArray(length)
        val random = Random()
        for (idx in buf.indices) buf[idx] = symbols[random.nextInt(symbols.size)]
        return String(buf)
    }
}